modules/benchmark/R/load_data.R

```diff
@@ -26,13 +26,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    key   <- get_key("https://bd-api.ncsa.illinois.edu",username,password)
+    token <- get_token("https://bd-api.ncsa.illinois.edu",key)
+    #output_path = "/fs/data3/jam2767/veg_test" where are we putting converted file?
+    converted.data.path <- convert_file(url = "https://bd-api.ncsa.illinois.edu", input_filename = data.path, 
+                 output = "csv", output_path, token)
```

Comment:
note from email with Yan, I had output and output path flipped in order in my code, which I sent you. You should probably refer to all args by name to be safe.

Also, all the BrownDog R function calls are missing their namespace, which you'll want to add.